specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2021-09-01-preview/agfood.json

```diff
@@ -48,8 +65,8 @@
           }
         },
         "x-ms-examples": {
-          "Extensions_Create": {
-            "$ref": "./examples/Extensions_Create.json"
+          "Extensions_CreateOrUpdate": {
+            "$ref": "./examples/Extensions_CreateOrUpdate.json"
```

Comment:
>Extensions_CreateOrUpdate [](http://example.com/codeflow?start=32&length=25)

Is this added?

Reply:
Added it in [this commit](https://github.com/Azure/azure-rest-api-specs/pull/20363/commits/424343746ed751f7bae17ff5b4cd1b896a5f84ef)